Makefile.dep

```diff
@@ -520,6 +520,9 @@ ifneq (,$(filter gnrc_pktbuf, $(USEMODULE)))
   ifeq (,$(filter gnrc_pktbuf_%, $(USEMODULE)))
     USEMODULE += gnrc_pktbuf_static
   endif
+  ifeq (gnrc_pktbuf_cmd,$(filter gnrc_pktbuf_%, $(USEMODULE)))
+    USEMODULE += gnrc_pktbuf_static
```

Comment:
wouldn't this pull in `gnrc_pktbuf_static`, even if, e.g., `gnrc_pktbuf_malloc` was set in the `USEMODULES`?

Reply:
nope. `$(filter gnrc_pktbuf_%, $(USEMODULE))` pulls out *all* modules that start with `gnrc_pktbuf_` so this `if*eq*` only is true when that filter returns only `gnrc_pktbuf_cmd`.

Reply:
alright :+1: